metaflow/plugins/argo/argo_workflows.py

```diff
@@ -3445,6 +3462,37 @@ def pvc_volumes(self, pvcs=None):
             )
         return self
 
+    def ephemeral_volume_claims(self, claims=None):
+        """
+        Create and attach Ephemeral Volume Claims as volumes.
+
+        Parameters:
+        -----------
+        claims: Optional[Dict]
+            a dictionary of ephemeral volumes name's to the paths they should be mounted to. e.g.
+            {"claim-1": {"path": "/mnt/path1", "spec": {"storageClassName": "my-claim"}, "metadata": {"labels": ["abc123"]}}}
+        """
+        if claims is None:
+            return self
+        if "volumes" not in self.payload:
+            self.payload["volumes"] = []
+        for name, values in claims.items():
+            self.payload["volumes"].append(
+                {
+                    "name": name,
+                    "ephemeral": {
+                        "volumeClaimTemplate": {
+                            "metadata": values.get("metadata", None),
+                            "spec": {
+                                **values.get("spec", {"abc": "def"}),
+                                **VOLUME_CLAIM_TEMPLATE_DEFAULTS,
```

Comment:
wouldn't the template defaults override user specified values in this order?

Reply:
Good callout, done